components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,767 @@
+    }
+
+    const teamTypeLabel = (type: string) => {
+        return getPlan(type)?.name;
+    }
+
+    return (<Modal visible={true} onClose={props.onClose}>
+        <h3 className="pb-2">New Team</h3>
+        <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-4 space-y-2">
+            <p className="pb-4 text-gray-500 text-base">Create a team and add team members.</p>
+
+            <div className="flex flex-col space-y-2">
+                <label htmlFor="type" className="font-medium">Team</label>
+                <select name="type" value={type} className="rounded-md w-full border-2 border-gray-400"
+                    onChange={(e) => setType(e.target.value)}>
+                    {props.types.map(type => (
+                        <option key={`type-option-${type}`} value={type}>{teamTypeLabel(type)}</option>
+                    ))}
+                </select>
+            </div>
+
+            <div className="flex flex-col space-y-2">
+                <label htmlFor="quantity" className="font-medium">Members</label>
+                <select name="quantity" value={quantity} className="rounded-md w-full border-2 border-gray-400"
+                    onChange={(e) => setQuantity(e.target.value as any)}>
+                    {Array(20).fill(1).map((_, index) => index + 1).map(n => (
+                        <option key={`quantity-${n}`} value={n}>{n}</option>
+                    ))}
+                </select>
+            </div>
+
+            <div className="flex rounded-md bg-gitpod-kumquat-light p-3 mt-2">
+                <img className="w-4 h-4 mx-2 my-auto" src={exclamation} />
+                <span className="text-red-600">Total: {expectedPrice} per month</span>
+            </div>
+
+        </div>
+        <div className="flex justify-end mt-6">
+            <button className={"ml-2"} onClick={() => props.onBuy(getPlan(type), quantity)}>Continue to Billing</button>
+        </div>
+    </Modal>);
+}
+
+function ManageTeamModal(props: {
+    slots: Slot[],
+    slotInputHandler: SlotInputHandler,
+    onClose: () => void,
+}) {
+
+    const [slots, setSlots] = useState<Slot[]>([]);
+
+    useEffect(() => {
+        const activeSlots = props.slots.filter(s => s.state !== 'cancelled');
+        setSlots(activeSlots);
+    }, [props.slots])
+
+    return (<Modal visible={true} onClose={props.onClose}>
+        <h3 className="pb-2">Manage Team</h3>
+        <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-4 space-y-2">
+            <p className="pb-4 text-gray-500 text-base">Add members using their username.</p>
+
+            {slots.map((slot, index) => {
+                return (
+                    <Slot key={slot.id} slot={slot} inputHandler={props.slotInputHandler} />
+                )
+            })}
+        </div>
+        <div className="flex justify-end mt-6">
+            <button className={"ml-2"} onClick={() => props.onClose()}>Close</button>
```

Comment:
**suggestion**: Close button can always be a secondary button. Some button variants have been introduced in [`index.css`](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/index.css#L37-L51), see `L37`-`L51`.

```suggestion
            <button className={"ml-2 secondary"} onClick={() => props.onClose()}>Close</button>
```